### ISSUE TITLE
Namespace info for partial completions is missing

### ISSUE THREAD

**Issue by cpitclaudel** [FStarLang/FStar#945]
F* is missing namespace information for some partial completions.  For example:

* `printf "module Test\nopen FStar.List\n" > test.fst`

* `fstar.exe test.fst --in`
```
#push 1 0
module Test
open FStar.List
#end #done-ok #done-nok
#completions List.append #
```

* **Output:**

```
11  List.append 
11  List.append_assoc 
11  List.append_cons_l 
11  List.append_count 
11  List.append_count_forall 
11  List.append_eq_nil 
11  List.append_eq_singl 
11  List.append_inv_head 
11  List.append_inv_tail 
11  List.append_l_cons 
11  List.append_l_nil 
11  List.append_length 
11  List.append_mem 
11  List.append_mem_forall 
11  List.append_nil_l 
11  List.append_sorted 
```

* **Expected:**

```
11 FStar List.append 
11 FStar List.append_assoc 
11 FStar List.append_cons_l 
11 FStar List.append_count 
11 FStar List.append_count_forall 
11 FStar List.append_eq_nil 
11 FStar List.append_eq_singl 
11 FStar List.append_inv_head 
11 FStar List.append_inv_tail 
11 FStar List.append_l_cons 
11 FStar List.append_l_nil 
11 FStar List.append_length 
11 FStar List.append_mem 
11 FStar List.append_mem_forall 
11 FStar List.append_nil_l 
11 FStar List.append_sorted 
```

The correct output can be seen when completing `append` instead of `list.append`:

```
6 FStar.List.Tot.Base append 
6 FStar.List.Tot.Properties append_assoc 
6 FStar.List.Tot.Properties append_cons_l 
6 FStar.List.Tot.Properties append_count 
6 FStar.List.Tot.Properties append_count_forall 
6 FStar.List.Tot.Properties append_eq_nil 
6 FStar.List.Tot.Properties append_eq_singl 
6 FStar.List.Tot.Properties append_inv_head 
6 FStar.List.Tot.Properties append_inv_tail 
6 FStar.List.Tot.Properties append_l_cons 
6 FStar.List.Tot.Properties append_l_nil 
6 FStar.List.Tot.Properties append_length 
6 FStar.List.Tot.Properties append_mem 
6 FStar.List.Tot.Properties append_mem_forall 
6 FStar.List.Tot.Properties append_nil_l 
6 FStar.List.Tot.Properties append_sorted 
```

**Comment by cpitclaudel** [comment]
Examples updated for `--ide`:

```
$ bin/fstar.exe test.fst --ide
{"query-id": "0", "query": "push", "args": {"kind": "full", "code": "module Test\nopen FStar.List", "line":1, column:0}}
{"kind":"response","query-id":"0","status":"success","response":[]}
{"query-id": "1", "query": "autocomplete", "args": {"partial-symbol":"List.append"}}
{"kind":"response","query-id":"1","status":"success","response":[[11,"","List.append"],[11,"","List.append_assoc"],[11,"","List.append_cons_l"],[11,"","List.append_count"],[11,"","List.append_count_forall"],[11,"","List.append_eq_nil"],[11,"","List.append_eq_singl"],[11,"","List.append_inv_head"],[11,"","List.append_inv_tail"],[11,"","List.append_l_cons"],[11,"","List.append_l_nil"],[11,"","List.append_length"],[11,"","List.append_length_inv_head"],[11,"","List.append_length_inv_tail"],[11,"","List.append_mem"],[11,"","List.append_mem_forall"],[11,"","List.append_nil_l"],[11,"","List.append_sorted"]]}
```

**Comment by cpitclaudel** [comment]
Turns out this wasn't an easy bug. The fix for #946 also covers this
